src/registrar/assets/js/get-gov.js

```diff
@@ -154,9 +154,6 @@ function _checkDomainAvailability(el) {
   fetchJSON(`available/?domain=${el.value}`, callback);
 }
 
-/** Call the API to see if the domain is good. */
-const checkDomainAvailability = debounce(_checkDomainAvailability);
```

Comment:
Previously every time we typed a domain into the field in the alternative domains, we would check every single field  before clicking "save and continue". We now don't need to check every single time we type into a field, and only check when the "check availability" button is clicked so we don't need debounce